packages/js/settings-editor/typings/global.d.ts

```diff
@@ -0,0 +1,53 @@
+type SettingField = {
+    title?: string;
+    type: string;
+    id: string;
```

Comment:
Some won't have id's, like `sectionend`, so this will need to be optional

Reply:
Gotcha. Changed it to be optional: https://github.com/woocommerce/woocommerce/pull/52692/files#diff-3ac9817d2ebb47b820627e2ad030a3579e24a74925d7151e90a63e7ac5135629R4